scripts/helpers/db-command-helpers.ts

```diff
@@ -126,6 +127,8 @@ export const runResetQueries = async (
       // When owner is superuser - this is not needed, but in deployed environments and for test databases owner is a regular user and also needs such grant.
       `GRANT ${dbGqlRole} TO ${dbOwner};`,
       createEnvOwner ? `GRANT ${dbGqlRole} TO ${dbEnvOwner};` : '',
+      // By granting owner_role to the root role, we can then create the database while settin owner_role as the owner.
+      pgRoot ? `GRANT ${dbOwner} TO ${pgRoot}` : '',
```

Comment:
This seems strange. Shouldn't the root user have access to the whole postgres server anyway?

Reply:
In FSPGs that's not necessarily the case and the database creation would fail without this grant.